specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2019-09-01/keyvault.json

```diff
@@ -1208,6 +1209,7 @@
         "softDeleteRetentionInDays": {
           "type": "integer",
           "format": "int32",
+          "default": 90,
```

Comment:
Do you really want a default for PATCH? What if it's already set to something else - say, 42 - and they later want to update it to 90. If they set the value to 90, generated clients may elide the default value and so 42 never gets change. Like other model properties for PATCH, maybe there shouldn't be a default here since you can't be sure how the generated client might treat the default value.

Reply:
Fixed it